File: src/main/b-client/check_db.js
const Database = require('better-sqlite3');

try {
    const db = new Database('./sqlite/b_client_secure.db');

    console.log('=== B-Client Database Check ===');

    // Check user_cookies table
    const cookies = db.prepare('SELECT user_id, username, create_time FROM user_cookies ORDER BY create_time DESC').all();
    console.log(`\nFound ${cookies.length} cookies in user_cookies table:`);
    cookies.forEach((cookie, index) => {
        console.log(`${index + 1}. user_id: "${cookie.user_id}", username: "${cookie.username}", created: ${cookie.create_time}`);
    });

    // Check user_accounts table
    const accounts = db.prepare('SELECT user_id, username, website, account FROM user_accounts ORDER BY create_time DESC').all();
    console.log(`\nFound ${accounts.length} accounts in user_accounts table:`);
    accounts.forEach((account, index) => {
        console.log(`${index + 1}. user_id: "${account.user_id}", username: "${account.username}", website: "${account.website}", account: "${account.account}"`);
    });

    db.close();
} catch (error) {
    console.error('Error checking database:', error);
}

File: src/main/b-client-new/check_db.py
import sqlite3

# Connect to database
conn = sqlite3.connect('instance/b_client_secure.db')
cursor = conn.cursor()

# Check tables
cursor.execute("SELECT name FROM sqlite_master WHERE type='table'")
tables = cursor.fetchall()
print('Tables:', tables)

# Check user_accounts count
cursor.execute("SELECT COUNT(*) FROM user_accounts")
count = cursor.fetchone()[0]
print('user_accounts count:', count)

# Check user_cookies count
cursor.execute("SELECT COUNT(*) FROM user_cookies")
count = cursor.fetchone()[0]
print('user_cookies count:', count)

# Show all user_accounts records
cursor.execute("SELECT user_id, username, website, logout FROM user_accounts")
records = cursor.fetchall()
print('user_accounts records:', records)

# Show all user_cookies records
cursor.execute("SELECT user_id, username FROM user_cookies")
records = cursor.fetchall()
print('user_cookies records:', records)

conn.close()
